sections/footer.liquid

```diff
@@ -14,7 +14,29 @@
 <noscript>{{ 'component-rte.css' | asset_url | stylesheet_tag }}</noscript>
 <noscript>{{ 'disclosure.css' | asset_url | stylesheet_tag }}</noscript>
 
-<footer class="footer color-{{ section.settings.color_scheme }} gradient">
+{%- style -%}
+  .footer {
+    margin-top: {{ section.settings.margin_top | times: 0.75 | round: 0 }}px;
```

Comment:
Here is there a reason why the approach is different than for the padding where it uses `calc({{ section.settings.padding_top }}px * 0.75)` ?

Reply:
Not really - I just thought of this approach this time.  Which do you prefer?  I'm thinking this might be better for two reasons.

1. All the calculation is done within the Liquid, which might be clearer visually.
2. No `calc` happening in CSS.  It's _**technically**_ more consistently performant to do this sort of thing server-side rather than client-side, though it probably doesn't actually make a difference.

I'll probably change the others to use this approach again if we decide we like it better.

Reply:
Yeah liquid calculation sounds like a good option and removes the extra work on the CSS side 👌